src/Jobs/AfterImportJob.php

```diff
@@ -36,6 +39,11 @@ public function __construct($import, Reader $reader)
 
     public function handle()
     {
+        // Determine if the batch has been cancelled...
+        if ($this->batch()->cancelled()) {
```

Comment:
This would break if ShouldBatch isn't implemented and user would use the old queueing